### DISCUSSION TITLE
우리의 프로젝트 장소!!

### DISCUSSION THREAD

**Discussion by jinux127** [heading1/WYLSBingsu#21] [Show and tell]
<img width="237" alt="스크린샷 2022-06-13 오후 10 25 30" src="https://user-images.githubusercontent.com/59363543/173363984-e0bf37cc-ba76-43d0-857e-b424367517b6.png">

저희가 짧다면 짧은 시간동안 진행할 프로젝트 장소입니다~~ 같이 지지고 볶아봐용! 😆

**Comment by jinux127** [comment]
ㅎㅎ..맵꾸미기 재밌따!!!..

**Reply by sokojh** [reply]
ㅋㅋㅋㅋ 손재주가 좋으시네요. 저는 저렇게 이쁘게 생각이 안나던데
